src/Contract/Operation/Reductionable.php

```diff
@@ -22,9 +23,13 @@ interface Reductionable
      *
      * @see https://loophp-collection.readthedocs.io/en/stable/pages/api.html#reduction
      *
-     * @param mixed $initial
+     * @template V
+     * @template W
```

Comment:
I don't understand why this is different from [Reduceable](https://github.com/loophp/collection/blob/master/src/Contract/Operation/Reduceable.php#L26).

Same of ScanLeftable and ScanRightable, shouldn't they all be the same?

Reply:
You're right, there was an inconsistency, I just fixed it.

However, `ScanLeft` and `ScanRight` might return the initial value if the Iterator is empty, this is why it might return `V` or `W`.